source/docs/magento2/page-builder.md

```diff
@@ -0,0 +1,138 @@
+- **server side data conversion** will parse Magento HTML response to extract rich structured data exposed in GraphQL
+- **client side React components** will display the content using existing components, from data fetched from GraphQL
+
+<blockquote class="wip">
+**Work In Progress:** if you need details right now, please [contact us](mailto:contact@front-commerce.com). We will make sure to answer you in a timely manner.
+</blockquote>
+
+## Supported content types
+
+### Currently supported
+
+We currently support these content types in a basic way:
+- ButtonItem (`button-item`)
+- Buttons (`buttons`)
+- Column (`column`)
+- ColumnGroup (`column-group`)
+- Divider (`divider`)
+- Heading (`heading`)
+- Row (`row`)
+- Text (`text`)
+
+We'll support more options in the next few weeks.
+
+## Upcoming
+
+Our 2.12 focus will be to:
+- expose all possible data in GraphQL for supported content types
+- honor the most common used content type options and content types
+
+It should allow merchants to build advanced page layouts using native Magento features.
+
+Let us know if you have specific needs.
+
+## Styles
+
+Front-Commerce supports custom styles from the *Advanced* Magento settings of all content types:
+
+![Advanced styles Magento settings](./assets/page-builder-advanced-styles.jpg)
+
+## Extend the Page Builder
+
+You can extend existing Page Builder content types, or register new ones specific to your projects. To do so, there are 2 extension points: UI and GraphQL data resolution.
+
+### Customize UI components
+
+<blockquote class="wip">
+**Work In Progress:** if you need details right now, please [contact us](mailto:contact@front-commerce.com). We will make sure to answer you in a timely manner.
+</blockquote>
+
+- override `theme/modules/WysiwygV2/MagentoWysiwyg/PageBuilder/_appComponents.scss` to register your custom styles
+- override `theme/modules/WysiwygV2/MagentoWysiwyg/PageBuilder/appComponentsMap.js` to register new components (or override [existing ones](https://gitlab.com/front-commerce/front-commerce/blob/main/src/web/theme/modules/WysiwygV2/MagentoWysiwyg/PageBuilder/index.js))
+
+<!-- Override GraphQL fragment too (not yet externalized in a specific fragment FC code) -->
+
+### Expose content types data in GraphQL
+
+<blockquote class="wip">
+**Work In Progress:** if you need details right now, please [contact us](mailto:contact@front-commerce.com). We will make sure to answer you in a timely manner.
+</blockquote>
+
+The `PageBuilder` loader allows you to register new content types.
+
+First, you must define your content type. Content types must extend the `ContentType` class (see below). The `name` should match Magento's content type identifier and the `extractData` method can be used to return structured data to be exposed as GraphQL fields for this type.
+
+```
+import ContentType from "server/modules/magento2/wysiwyg/loaders/content-types/ContentType";
+
+export default class Foo extends ContentType {
+  name = "foo";
```

Comment:
it would be nice to find a realistic (but not too complicated example)

(can be made in a follow up as the page is WIP anyway)

Reply:
Yes, that's a good idea! I'll update it later.